src/js/engagement_view/src/components/engagementView/sidebar/tables/lensTable/selectLens.tsx

```diff
@@ -11,21 +10,19 @@ import { SelectLensProps } from "types/LensAndNodeTableTypes";
 export function SelectLens(props: SelectLensProps) {
     const classes = useStyles();
 
-    const [btnClass, setBtnClass] = useState(false);
-
     return (
-        <TableRow className={classes.tableRow} key={props.uid}>
+        <TableRow
+            hover
+            key={props.uid}
+            onClick={() => {
+                props.setLensTableState(props.uid);
+                props.setLens(props.lens);
+            }}
+            selected={props.selectedId === props.uid}
+            className={classes.tableRow}
+        >
             <TableCell component="th" scope="row" align="left">
-                <Button
-                    // className={classes.lensName}
-                    className={btnClass ? "btnClass clicked" : "btnClass"}
-                    onClick={() => {
-                        props.setLens(props.lens);
-                        btnClass ? setBtnClass(false) : setBtnClass(true);
-                    }}
-                >
-                    {props.lens_type + " :\t\t" + props.lens}
-                </Button>
+                {props.lens_type + " :\t\t" + props.lens}
```

Comment:
could you leave a comment on what does :\t\t do

Reply:
Dead and unnecessary code - removed.